app/src/main/java/com/pvp/app/ui/screen/friends/FriendsViewModel.kt

```diff
@@ -180,6 +194,55 @@ class FriendsViewModel @Inject constructor(
             )
         }
     }
+
+    fun getMutualFriends(friendEmail: String) {
+        viewModelScope.launch {
+            val friendObject = friendService
+                .get(friendEmail)
+                .firstOrNull()
+
+            val mutualFriendsList =
+                friendObject?.friends?.map { it.email }
```

Comment:
**All cases:**
- If there are properties in the chained call, they go within the same line (until no longer fits).
- If there is one method in whole chained call, it stays within the same line (until no longer fits).
- If there are multiple methods in whole chained call, each method is split into new line. If there are property calls in between method calls, these properties stay in the same line as the last called method.

**Here would be:**
```kotlin
val friends = friendObject?.friends
    ?.map { it.email }
    ?.intersect(
        userFriends.value
            .map { it.user.email }
            .toSet()
    )
```